src/service/exercise/ExerciseService.ts

```diff
@@ -7,16 +7,30 @@ export class ExerciseService {
   public async getAllExercisesService(page, count, hiringProcessId) {
     const exerciseRepository = getRepository(Exercise)
     const result = await exerciseRepository.createQueryBuilder()
-    .select("exercise")
-    .from(Exercise, "exercise")
-    .where("exercise.hiringProcess = :id", { id: hiringProcessId })
-    .leftJoinAndSelect("exercise.hiringProcess", "hiringProcess")
-    .skip(page)
-    .take(count)
-    .getMany()
+      .select("exercise")
+      .from(Exercise, "exercise")
+      .where("exercise.hiringProcess = :id", { id: hiringProcessId })
+      .leftJoinAndSelect("exercise.hiringProcess", "hiringProcess")
+      .skip(page)
+      .take(count)
+      .getMany()
     if (result.length === 0) {
       throw new HttpError(message.NOT_FOUND, HttpStatusCode.NOT_FOUND)
     }
     return result
   }
-}
\ No newline at end of file
+
+  public async editType(id, type) {
+    const exerciseRepository = getRepository(Exercise)
+    const exercise = await exerciseRepository.findOne(id)
+    if (!exercise) {
+      throw new HttpError('Exercise not found with: ' + id, HttpStatusCode.BAD_REQUEST)
```

Comment:
Aqui é pura chatisse mesmo, não precisa mudar, só queria mostra caso vocês ainda não conheçam. 

Seria possível utilizar [template strings](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Template_literals):

```suggestion
      throw new HttpError(`Exercise not found with: ${id}`, HttpStatusCode.BAD_REQUEST)
```

Nesse caso não faz muita diferença, mas quando temos que interpolar/concatenar várias variáveis em uma string, as template strings podem ser úteis.